src/main/js/components/dropdowns/autocomplete.js

```diff
@@ -59,9 +59,30 @@ function init() {
       }
       return;
     }
-    const url =
-      e.getAttribute("autoCompleteUrl") + "?value=" + encodeURIComponent(word);
-    fetch(url)
+
+    const url = e.getAttribute("autoCompleteUrl");
+
+    const depends = e.getAttribute("fillDependsOn");
+    const q = qs(e).addThis();
+    if (depends && depends.length > 0) {
+      depends.split(" ").forEach(
+        TryEach(function (n) {
+          q.nearBy(n);
+        }),
+      );
+    }
+
+    const queryString = q.toString();
+    const idx = queryString.indexOf("?");
+    const parameters = queryString.substring(idx + 1);
```

Comment:
copied from `hudson-behaviour.js` for the `doCheck` functionality. 

It uses query string by default and then if the method is post it converts to parameters, (which is the default)

I didn't see any reason for there to need to be an option for get, I'm not aware of it ever being used for the doChecks